components/hab/src/main.rs

```diff
@@ -868,6 +873,21 @@ fn auth_token_param_or_env(ui: &mut UI, m: &ArgMatches) -> Result<String> {
     }
 }
 
+/// Check to see if the user has passed in a CTL_SECRET param. If not, check the
```

Comment:
I see this checking the `CTL_SECRET_ENVVAR` env var and the CLI config, but is there code I'm missing to "[c]heck to see if the user has passed in a CTL_SECRET param", or does this comment need updating?

Reply:
Just a bad comment. Fixed it!